api/layer/neofs/neofs.go

```diff
@@ -33,9 +33,6 @@ type PrmContainerCreate struct {
 	// Token of the container's creation session. Nil means session absence.
 	SessionToken *session.Token
 
-	// Time when container is created.
-	Time time.Time
```

Comment:
If we get rid of the parameter, but still require the creation time to be set (`time.Now()`), let's describe it in the documentation for the `NeoFS.CreateContainer` interface.

Similar for `tokens.PrmObjectCreate`.